pyaedt/generic/general_methods.py

```diff
@@ -161,32 +161,34 @@ def _remote_dict_conversion(args):
 
 def _log_method(func, new_args, new_kwargs):
     if str(func.__name__)[0] != "_":
-        line_begin = "\n                                    Implicit Arguments : "
-        line_begin2 = "\n                                    Explicit Arguments : "
+        line_begin = "    Implicit Arguments: "
+        line_begin2 = "    Explicit Arguments: "
+        message = []
```

Comment:
```suggestion
        messages = []
```
@Alberto-DM You could rename it as a plural because at the end it will contain several messages and you iterate other them.